docs/_spec/05-classes-and-objects.md

```diff
@@ -925,18 +925,15 @@ First, some terminology and notational conventions:
 - We use `<...>` for syntactic constructs that in some circumstances might be empty.
 For instance, `<value-params>` represents one or more parameter lists `(´\mathit{ps}_1\,´)...(´\mathit{ps}_n´)` or nothing at all.
 - Enum classes fall into two categories:
-  - _parameterized_ enum classes have at least one of the following:
-     - a type parameter section, denoted as `[´\mathit{tps}\,´]`;
-     - one or more (possibly empty) parameter sections, denoted as `(´\mathit{ps}_1\,´)...(´\mathit{ps}_n´)`.
-  - _unparameterized_ enum classes have no type parameter sections and no parameter sections.
+  - _parameterized_ enum classes have at least one or more (possibly empty) term parameter clauses, denoted as `(´\mathit{ps}_1\,´)...(´\mathit{ps}_n´)`.
+  - _unparameterized_ enum classes have no term parameter clauses, but may optionally have a type parameter clause, denoted as `[´\mathit{tps}\,´]`.
```

Comment:
That does not appear to be true:
```
scala> enum Foo { case Bar[T] }
-- [E004] Syntax Error: --------------------------------------------------------
1 |enum Foo { case Bar[T] }
  |                ^^^
  |                A case class must have at least one parameter list
  |
  | longer explanation available when compiling with `-explain`
1 error found
```

Reply:
this paragraph is about the base enum class, not a class case - hmm these terms are confusing